### ISSUE TITLE
PHP Warning:  Undefined array key

### ISSUE THREAD

**Issue by ghost** [pjv/nostrtium#4]
```
2023/06/11 09:51:43 [error] 69708#69708: *1060 FastCGI sent in stderr: "PHP message: PHP Warning:  Undefined array key 2 in /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-content/plugins/nostrtium/vendor/swentel/nostr-php/src/Relay/CommandResult.php on line 38; PHP message: PHP Fatal error:  Uncaught ErrorException: Undefined variable $post_id in /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-content/plugins/nostrtium/classes/class-nostrtium.php:106
Stack trace:
#0 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-content/plugins/nostrtium/classes/class-nostrtium.php(106): Phrity\Util\ErrorHandler->Phrity\Util\{closure}()
#1 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-includes/class-wp-hook.php(308): Nostrtium->maybe_publish_post_to_nostr()
#2 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters()
#3 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-includes/plugin.php(517): WP_Hook->do_action()
#4 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-includes/post.php(5345): do_action()
#5 /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64et" while reading response header from upstream, client: 192.168.0.121, server: mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion, request: "POST /wp-admin/admin-ajax.php HTTP/1.0", upstream: "fastcgi://unix:/run/php/php8.2-fpm.sock:", host: "mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion", referrer: "http://mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/"
2023/06/11 09:53:15 [error] 69708#69708: *1118 FastCGI sent in stderr: "PHP message: PHP Warning:  Undefined array key "page" in /var/www/mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-content/plugins/nostrtium/classes/class-nostrtium.php on line 61" while reading response header from upstream, client: 192.168.0.121, server: mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion, request: "GET /wp-admin/options-general.php HTTP/1.0", upstream: "fastcgi://unix:/run/php/php8.2-fpm.sock:", host: "mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion", referrer: "http://mwaters7eu2pyfcz36oehlc3htnqarakd7meeyz4lvbubff64ethxzqd.onion/wp-admin/edit.php"
```

Seeing these errors in the nginx error log , despite everything working.

**Comment by pjv** [comment]
Hoping this is fixed with v0.7.2

**Comment by ghost** [comment]
No longer seeing this error , thanks @pjv
